internal/util/get/get.go

```diff
@@ -116,6 +121,23 @@ func (c Command) Run(ctx context.Context) error {
 		Outputs: []kio.Writer{inout},
 	}.Execute()
 
+	if c.IsDeploymentInstance {
+		pr := printer.FromContextOrDie(ctx)
+		pr.Printf("\nCustomizing package for deployment.\n")
+		hookCmd := hook.Executor{
+			PkgPath: c.Destination,
+		}
+		builtinHooks := []kptfilev1.Function{
+			{
+				Image: "builtins/gen-pkg-context",
```

Comment:
nit: how about const var since the `builtins/gen-pkg-context` is referred in other places as well"?

Reply:
Done.